apps/web/vercel.json

```diff
@@ -0,0 +1,8 @@
+{
+  "crons": [
+    {
+      "path": "/api/cron/calendar-cache-cleanup",
+      "schedule": "0 5 * * *"
```

Comment:
```suggestion
      // Every day At 05:00.” 
      "schedule": "0 5 * * *"
```

Reply:
A little hint to the eye

Reply:
I don't think we can put comments on naked json files

Reply:
<img width="628" alt="image" src="https://github.com/calcom/cal.com/assets/3504472/066320b8-7832-4d79-8ba3-259561c4f0f3">